docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>"A feature is introduced in TensorBoard v2.3 supports this use case" -&gt; "TensorBoard 2.3 supports this use case with tensorboard.data.experimental.ExperimentFromDev()"</p><p> </p><p>I don't know if we actually need to mention the version in the text (you check the version later in the notebook anyway). But maybe full information is better.</p><p> </p><p>Consider linking the first mention of "scalar logs" to the appropriate tutorial: <a href="https://www.tensorflow.org/tensorboard/scalars_and_keras" target="_blank">https://www.tensorflow.org/tensorboard/scalars_and_keras</a>. Some people might not think of the word Scalars for this.</p><p> </p><p>The first mention of TensorBoard.dev in note 2 should be a link to TensorBoard.dev.</p><p> </p><p>"filesystem to tensorboard.dev ..." -&gt; "filesystem to TensorBoard.dev"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done. Yep, let's be detailed and specific about version.